src/extension.js

```diff
@@ -460,6 +473,135 @@ async function onDidExecuteTerminalCommandHelper(event) {
 	}
 }
 
+// https://stackoverflow.com/questions/25245716/remove-all-ansi-colors-styles-from-strings
+function removeANSIcodesHelper(txt) {
+	const processedTxt = txt.replace(/[\u001b\u009b][[()#;?]*(?:[0-9]{1,4}(?:;[0-9]{0,4})*)?[0-9A-ORZcf-nqry=><]/g, '');
+	return processedTxt;
+}
+
+// https://stackoverflow.com/questions/20856197/remove-non-ascii-character-in-string
+function removeNonASCIICharsHelper(txt) {
+	const processedTxt = txt.replace(/[\x00-\x08\x0E-\x1F\x7F-\uFFFF]/g, '');
+	return processedTxt;
+}
+
+async function onDidExecuteShellCommandHelper(event) {
+	try {
+		// console.log('event.terminal', event.terminal);
+		// console.log('event.shellIntegration', event.shellIntegration);
+		// console.log('event.execution', event.execution);
+		if(event.execution.commandLine){
+			let time = Math.floor(Date.now() / 1000);
+			let command = event.execution.commandLine ? event.execution.commandLine.value : '';
+			let cwd = event.execution.cwd ? event.execution.cwd.path : '';
+			let outputStream = event.execution.read();
+			let output = '';
+			for await (let data of outputStream) {
+				data = data.toString();
+				data = removeANSIcodesHelper(data);
+				data = removeNonASCIICharsHelper(data);
+				// console.log('data:', data);
+				output += data;
+			}
+
+			// Regex pattern to remove ]633;...; or ]633;..., sequences with at most one character between ]633; and the next ; or ,
+			const weirdSeq1Regex = /\]633;[^;|^,]?;|\]633;[^;|^,]?,/g;
+
+			// Regex pattern to replace \x5c with /
+			const weirdSeq2Regex = /\\x5c/g;
+
+			// Regex pattern to match UIID
+			const uuidRegex = /[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}/g;
+
+			// Regex to remove ,\n with \n
+			const weirdSeq3Regex = /,\n|,\n\r|,\r|,\r\n/g;
```

Comment:
This was a bug which should be fixed soon https://github.com/microsoft/vscode/pull/224741